ogr/ogr_geometry.h

```diff
@@ -608,6 +608,9 @@ class CPL_DLL OGRGeometry
 
     OGRGeometry *SetPrecision(double dfGridSize, int nFlags) const;
 
+    virtual bool hasEmptyParts() const;
```

Comment:
It seems to me that there is a mix of styles regarding the name of methods: should they start with a capital letter or not?

Reply:
Yes, I had the same thought and can't really find the nominal pattern here... In most of the rest of the code base methods generally start with capital letter, but OGRGeometry has always had that mix